kpi/tests/api/v2/test_api_asset_counts.py

```diff
@@ -0,0 +1,261 @@
+from django.contrib.auth.models import User
+from django.urls import reverse
+from rest_framework import status
+
+from kpi.constants import PERM_VIEW_ASSET, PERM_VIEW_SUBMISSIONS
+from kpi.tests.base_test_case import BaseAssetDetailTestCase
+from kpi.urls.router_api_v2 import URL_NAMESPACE as ROUTER_URL_NAMESPACE
+
+
+class UsageAPITestCase(BaseAssetDetailTestCase):
+    fixtures = ['test_data']
+    URL_NAMESPACE = ROUTER_URL_NAMESPACE
+
+    def test_count_endpoint_owner(self):
+        self.client.login(username='someuser', password='someuser')
+        self.asset.content = {
+            'survey': [
+                {
+                    'type': 'select_one',
+                    'label': 'q1',
+                    'select_from_list_name': 'iu0sl99'
+                },
+            ],
+            'choices': [
+                {'name': 'a1', 'label': ['a1'], 'list_name': 'iu0sl99'},
+                {'name': 'a3', 'label': ['a3'], 'list_name': 'iu0sl99'},
+            ]
+        }
+        self.asset.save()
+        self.asset.deploy(backend='mock', active=True)
+        submissions = [
+            {
+                '__version__': self.asset.latest_deployed_version.uid,
+                'q1': 'a1',
+                '_submitted_by': 'anotheruser',
+                '_submission_time': '2022-09-07T13:21:33',
+            },
+            {
+                '__version__': self.asset.latest_deployed_version.uid,
+                'q1': 'a3',
+                '_submitted_by': '',
+                '_submission_time': '2022-09-12T16:31:33',
+            },
+            {
+                '__version__': self.asset.latest_deployed_version.uid,
+                'q1': 'a1',
+                '_submitted_by': '',
+                '_submission_time': '2022-09-12T17:31:33',
+            },
+        ]
+
+        self.asset.deployment.mock_submissions(submissions)
```

Comment:
I think these lines could be in `setUp()` since we are using always the same content in this class.